codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodec.java

```diff
@@ -378,10 +390,19 @@ void flush0(ChannelHandlerContext ctx) {
         flush(ctx);
     }
 
-    // Allow to override for testing
-    void onBytesConsumed(@SuppressWarnings("unused") ChannelHandlerContext ctx,
+    /**
+     * Return bytes to flow control.
+     * <p>
+     * Package private to allow to override for testing
+     * @param ctx The {@link ChannelHandlerContext} associated with the parent channel.
+     * @param stream The object representing the HTTP/2 stream.
+     * @param bytes The number of bytes to return to flow control.
+     * @return {@code true} if a frame has been written as a result of this method call.
+     * @throws Http2Exception If this operation violates the flow control limits.
+     */
+    boolean onBytesConsumed(@SuppressWarnings("unused") ChannelHandlerContext ctx,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "ctx". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1172)